src/lib/bcmp/bcmp_config.cpp

```diff
@@ -389,10 +389,15 @@ static void bcmp_process_del_response_message(bm_common_config_delete_key_respon
     vPortFree(keyprintbuf);
 }
 
-void bcmp_process_config_message(bcmp_message_type_t bcmp_msg_type, uint8_t* payload) {
+/*!
+    \return true if the caller should forward the message, false if the message was handled
+*/
+bool bcmp_process_config_message(bcmp_message_type_t bcmp_msg_type, uint8_t *payload) {
+    bool should_forward = false;
     do {
         bm_common_config_header_t * msg_header = reinterpret_cast<bm_common_config_header_t *>(payload);
-        if(msg_header->target_node_id != getNodeId()){
+        if (msg_header->target_node_id != getNodeId()) {
+            should_forward = true;
             break;
         }
```

Comment:
Instead of just dropping the packet, let's return true to indicate BCMP should forward the message.